frontend/tsconfig.json

```diff
@@ -14,8 +16,11 @@
     "isolatedModules": true,
     "noEmit": true,
     "jsx": "react-jsx",
-    "baseUrl": ".",
+    "baseUrl": "./src",
```

Comment:
The baseUrl should be the root directory of your project, not the src directory.The baseUrl should be "./".

The baseUrl setting is used to resolve relative imports. For example, if you have a file called app.ts in the src directory, and you import a file called components/Button.ts, the TypeScript compiler will look for the Button.ts file in the src/components directory.

If you set the baseUrl to "./src", then the TypeScript compiler will look for the Button.ts file in the src/src/components directory, which is not correct.

You can fix this by changing the baseUrl setting to "./". This will tell the TypeScript compiler to look for relative imports in the root directory of your project.